course_access_groups/models.py

```diff
@@ -30,11 +31,25 @@ class CourseAccessGroup(models.Model):
 
 class Membership(utils_models.TimeStampedModel):
     """
-    Student membership in a CourseAccessGroup.
+    Student membership in a Group.
     """
 
-    group = models.ForeignKey(CourseAccessGroup)
+    group = models.ForeignKey(Group)
     user = models.OneToOneField(
         get_user_model(),
         help_text='Student. A student can only be enrolled in a single Course Access Group.'
     )
+
+
+class Rule(utils_models.TimeStampedModel):
+    name = models.CharField(max_length=255, help_text='A description for this assignment rule.')
+    domain = models.CharField(max_length=255, db_index=True)
```

Comment:
Is this the domain name? Please add context like you did with `name` in `help_text`